pkg/vmprovider/providers/vsphere/vmprovider_vm_utils.go

```diff
@@ -49,6 +54,10 @@ func GetVirtualMachineClass(
 		return nil, errors.Wrap(err, msg)
 	}
 
+	if lib.IsNamespacedClassAndWindowsFSSEnabled() {
```

Comment:
Nit: maybe add a comment here indicating why we don't need to check the bindings etc.